model/registry/vulnerability.yaml

```diff
@@ -0,0 +1,120 @@
+groups:
+  - id: registry.vulnerability
+    prefix: vulnerability
+    type: attribute_group
+    display_name: Vulnerability
+    brief: >
+      The vulnerability fields describe information about a vulnerability.
+    attributes:
+      - id: category
+        type: string[]
+        stability: experimental
+        brief: >
+          The type of system or architecture that the vulnerability affects. These may be platform-specific
+          (for example, Debian or SUSE) or general (for example, Database or Firewall). For example
+          [Qualys vulnerability categories](https://qualysguard.qualys.com/qwebhelp/fo_portal/knowledgebase/vulnerability_categories.htm).
+        examples: ['Firewall']
+      - id: classification
+        type: string
+        stability: experimental
+        brief: >
+          The classification of the vulnerability scoring system. For example, [CVSS](https://www.first.org/cvss/specification-document).
+        examples: ['CVSS']
+      - id: description
+        type: string
+        stability: experimental
+        brief: >
+          The description of the vulnerability that provides additional context of the vulnerability.
+          For example [Common Vulnerabilities and Exposure CVE description](https://www.cve.org/ResourcesSupport/FAQs).
+        examples: ['A vulnerability in the firewall allows an attacker to bypass security controls.']
+      - id: enumeration
+        type: string
+        stability: experimental
+        brief: >
+          The type of identifier used for this vulnerability. For example, [Common Vulnerabilities and Exposure CVE enumeration](https://www.cve.org/ResourcesSupport/FAQs).
+        examples: ['CVE']
+      - id: id
+        type: string
+        stability: experimental
+        brief: >
+          The number portion of a vulnerability entry. It includes a unique identification number for the vulnerability.
+          For example, [Common Vulnerabilities and Exposure CVE ID](https://www.cve.org/ResourcesSupport/FAQs).
+        examples: ['CVE-2021-44228']
+      - id: reference
+        type: string
+        stability: experimental
+        brief: >
+          A resource that provides additional information, context, and mitigations for the identified vulnerability.
+        examples: ['https://www.cve.org/CVERecord?id=CVE-2021-44228']
+      - id: report_id
```

Comment:
Would there be a future case where `report` would be moved to a namespace where things like `url.full` could be added to link to the report, or other attributes?

Reply:
If this is the id of the scan, it would make sense to name `scan.id`

Reply:
@adrielp as we have agreed during one of semconv on having `id` always in it's own namespace, I will update this field